templates/repo/issue/filter_list.tmpl

```diff
@@ -15,7 +15,7 @@
 			<input type="text" placeholder="{{ctx.Locale.Tr "repo.issues.filter_milestone"}}">
 		</div>
 		<div class="divider"></div>
-		<a class="{{if not $.MilestoneID}}active selected {{end}}item" href="{{QueryBuild $queryLink "milestone" 0}}">{{ctx.Locale.Tr "repo.issues.filter_milestone_all"}}</a>
+		<a class="{{if not $.MilestoneID}}active selected {{end}}item" href="{{QueryBuild $queryLink "milestone" NIL}}">{{ctx.Locale.Tr "repo.issues.filter_milestone_all"}}</a>
```

Comment:
Why do we write `NIL` as uppercase? Shouldn't it work as `nil` too?

Reply:
Odd but do do have a few cases of lowercase `nil` in the templates:

```
templates/repo/file_info.tmpl:7:  {{if ne .NumLines nil}}
templates/repo/file_info.tmpl:12: {{if ne .FileSize nil}}
templates/repo/issue/view_content.tmpl:8:         {{ctx.AvatarUtils.Avatar nil 40}}
templates/repo/issue/view_content/comments.tmpl:21:         {{ctx.AvatarUtils.Avatar nil 40}}
templates/repo/diff/comments.tmpl:6:    <span class="avatar">{{ctx.AvatarUtils.Avatar nil}}</span>

Reply:
That's different. Official document and issue have explained.

Reply:
The real example is:

`{{$var := nil}}`, it fails. We have to use `{{$var := NIL}}`